README.md

```diff
@@ -103,41 +45,71 @@ graph LR
     E_1 -.-> E_0
 ```
 
-## Questions
+# How room are managed?
 
-> Will user who likes the same worlds will like eachothers? Will it work with few worlds?
+## User - Start Awa
 
-# How room are managed?
+```mermaid
+sequenceDiagram
+    participant oth as other room users
+    actor u as user
+    participant m as matchmaker
+    participant s as websocket server
+    participant f as firebase
+    u ->> u: GET room on disk
+    alt has room
+        u ->> s: connect to websocket server
+        alt connection not accepted
+            u ->> f: unsubscribe to room_id topic
```

Comment:
This 'alt' part is not clear for me, but it may be OK. I am probably just unaware of this part of the process.